server/public/styles.css

```diff
@@ -0,0 +1,18 @@
+.profile-picture {
+  width: 225px;
+  height: 225px;
+}
+
+@media only screen and (min-width: 725px) {
+  .text-align-center {
+    text-align: start;
+  }
+
+  .mobile-row {
+    display: flex;
+  }
```

Comment:
Things that are specifically for layout (including media queries) should go in `layout.css`.